src/unittest/extproc_test.cc

```diff
@@ -263,9 +263,9 @@ class segfault_job_t : public base_crash_job_t {
 
 private:
     static bool worker_fn(read_stream_t *, write_stream_t *) {
-        int bad_data = *reinterpret_cast<volatile int*>(NULL);
```

Comment:
Interesting. Which version of Clang was this with?

We compile with Clang frequently on both Linux and OS X, so it should generally work. This might have stopped working with a recent version of Clang, in which case we'll need to fix it for Linux anyway. Do you remember the error message that Clang gave?